manager/webserver.go

```diff
@@ -270,7 +272,8 @@ func StartWebServer(host string, httpPort int) {
 
 	log.Println("Starting web server")
 	server.RegisterOnShutdown(onShutdown)
-	botPool = bots.NewBotPool(4)
+	// start as many bot instances as cpu has cores
+	botPool = bots.NewBotPool(runtime.NumCPU())
```

Comment:
For a specific reason? Are the instances working on a specific core automatically? Or is it possible that all 4 instances use the same core? If so, why limit to the amount of cores the CPU has?

Reply:
Go concurrency automatically uses as much cores as possible.